### ISSUE TITLE
[Bug] - can't set Required?

### ISSUE THREAD

**Issue by Elaniobro** [Kocal/jsdoc-vuejs#364]
Awesome plugin, really liking how it works so far.

Not sure if this a bug, known or other, but I can't seem to set the `Required?` table cell in the `@vue-props` declaration.

```js
/**
 * @vue-prop {('cube-grid'|'three-bounce')} [name='cube-grid'] - optional='Yes' - true - name of loader to use
 * @vue-prop {Number} [size=30] - [optional='Yes'] - size of loading icons width
 * @vue-prop {String} [bgColor=#000] - required='Yes' - hexadecimal color value for loader
 * @vue-prop {String} [bgColor=#000] - [required='Yes'] - hexadecimal color value for loader
*/
```

Nothing seems to work, and `Required?` is always `No`

**Comment by Kocal** [comment]
Hi, and thanks for your words.

I think you need to use this to make a property required:
```js
/**
 * @vue-prop {String} bgColor - hexadecimal color value for loader
 */
```

Since it uses `bgColor` instead of `[bgColor=#000]`, the property is now seen as required.

**Comment by Elaniobro** [comment]
Thank you for your reply, how then would you set the default value, along side required?
`* @vue-prop {('cube-grid'|'three-bounce')} [name='cube-grid'] - name of loader to use`
<img width="621" alt="Screen Shot 2020-08-11 at 3 25 55 PM" src="https://user-images.githubusercontent.com/710847/89940090-f950bb00-dbe6-11ea-803d-1640f737fdbe.png">

`* @vue-prop {('cube-grid'|'three-bounce')} name - name of loader to use`
<img width="636" alt="Screen Shot 2020-08-11 at 3 24 42 PM" src="https://user-images.githubusercontent.com/710847/89940044-e938db80-dbe6-11ea-888c-b627d73568c2.png">

I can't seem to get both to work, the bug, or I am doing something very wrong. Thanks!

**Comment by Kocal** [comment]
How can a Vue props can be required if it has a default value?

**Comment by Elaniobro** [comment]
@Kocal fair enough, but the documentation is then confusing since `Required?` says `no`. Would it not make more sense to then void the value e.g. `-`since the `Default Value` is present? Similar to how its show when `Required?` is `Yes`?

**Comment by Programina** [comment]
I will be honest here and admit that I also find the documentation might benefit from being a little clearer on what part of that syntax designates what. I am currently doing a lot of trial and error to get both the name of the prop and the required state right and haven't managed both at once yet.
